packages/utils/src/types/json-parser.ts

```diff
@@ -1,4 +1,4 @@
-import { ProblemLocation } from './problems';
+import { ProblemLocation } from './problem-location';
 
 export interface IJSONLocationOptions {
```

Comment:
Not blocking for this PR, but what do you think about starting to group types with util implementation instead of in a separate folder? E.g. having a `src/json` folder with `parser` and `types`?

Reply:
SGTM, we will have to maintain both things until the next breaking.

Reply:
Yeah, I'd probably hold off on doing it at all until the next breaking opportunity.